enclave-tls/src/enclave_quotes/sgx-ecdsa/verify_evidence.c

```diff
@@ -42,6 +42,13 @@ enclave_quote_err_t sgx_ecdsa_verify_evidence(enclave_quote_ctx_t *ctx,
 	ETLS_DEBUG("quote size is %d, quote signature_data_len is %d\n",
 		   quote_size, pquote->signature_data_len);
 
+	// First verify the hash value
+	int result = memcmp(hash, pquote->report_body.report_data.d, SHA256_HASH_SIZE);
```

Comment:
The size of hash is not fixed in length. Here we lost a hash_len parameter passed from up to down. tls_wrapper_verify_certificate_extension() requires to add it. @YangLiang3 and @Ruoyu-y What are your opinions?

Reply:
Now during creation of hash, we only provide a fixed length uint_8 array to hold that. So shall we calculate the hash size inside tls_wrapper_verify_certificate_extension() and then pass to the verify func, or we just calculate the size directly inside the verify func before we compare the buffer?

Reply:
See tls_wrappers/wolfssl/un_negotiate.c. cert_algo is configurable, and hash_size is calculated there.

Reply:
@jiazhiguang it's reasonable